packages/tabs/demo/tabs.stories.mdx

```diff
@@ -2,18 +2,20 @@ import { Meta, ArgsTable, Canvas, Story, Markdown } from '@storybook/addon-docs'
 import { useArgs } from '@storybook/client-api';
 import { TabsContainer } from '@zendeskgarden/container-tabs';
 import { TabsStory } from './stories/TabsStory';
+import { ITEMS } from './stories/data';
 import README from '../README.md';
 
 <Meta
   title="Packages/Tabs"
   component={TabsContainer}
   args={{
     as: 'hook',
-    tabs: 3
+    tabs: 3,
+    items: ITEMS
   }}
   argTypes={{
     as: { options: ['container', 'hook'], control: 'radio', table: { category: 'Story' } },
-    tabs: { control: { type: 'range', min: 1, max: 5 }, table: { category: 'Story' } }
+    tabs: { table: { category: 'Story' } }
```

Comment:
Does the `tabs` control do anything?

The `orientation` control should be fixed to select 'horizontal' by default – corresponding with the API (and similar to `selection.stories.mdx`).

Reply:
Good call, it should be `tabs`. I must've copied it over from `selection` and forgot to rename it.

Actually, based on the new API, this arg doesn't really need to change, so if it's preferred I can switch it back to the `range` control.

I didn't touch `orientation` in either storybook demo but I can add that in.

Reply:
I think your new `items` array displaces `tabs` (range or not) entirely, right?

Reply:
Yep. But the container API accepts any value, so indices would still work. My intent was to make the data input clearer with `items`.